### ISSUE TITLE
[BUG] VSCode extension - [error] TypeError: Cannot read properties of null (reading 'Symbol(traceUrl)')

### ISSUE THREAD

**Issue by bjerkins** [microsoft/playwright#24342]
**Context:**
- Playwright Version: 1.35.1
- Operating System: Mac
- Node.js version: 18.16.0
- Visual Studio Code version: 1.80.1
- Playwright for VSCode extension version: 1.0.15

**Describe the bug**

Hello! 

I'm trying to view/run my Playwright tests in vscode, but the extension claims: "No tests have been found in this workspace yet"

![screenshot 2023-07-21 at 12 49 46](https://github.com/microsoft/playwright/assets/2454547/ca2f00d7-40bc-4783-887a-bb36c90bd47a)

When I look at the logs I can see that the extension is throwing an error:

```
[error] TypeError: Cannot read properties of null (reading 'Symbol(traceUrl)')
    at Ve._treeItemSelected (/Users/user/.vscode/extensions/ms-playwright.playwright-1.0.15/out/extension.js:109:453)
    at i.value (/Users/user/.vscode/extensions/ms-playwright.playwright-1.0.15/out/extension.js:91:4664)
    at p.w (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:1902)
    at p.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:2119)
    at St._poll (/Users/user/.vscode/extensions/ms-playwright.playwright-1.0.15/out/extension.js:110:727)
```

Any ideas ?

**Comment by aslushnikov** [comment]
@bjerkins any chance you can share a small repo that we can clone that reproduces this behavior?

**Comment by bjerkins** [comment]
Hey @aslushnikov , thanks for the response. I'll try to reproduce with a fresh repository and see if I can reproduce the error. We have a monorepo where this one project doesn't play well with the extension, the others do perfectly. My hope was that the error could give some clues that could point me to the right direction 😃

**Comment by aslushnikov** [comment]
This is fixed in a VSCode extension; the fix will be live in the upcoming extension push.

Thanks for filing!

**Comment by sean-shirazi** [comment]
I ran into the same issue. I applied the change in #364 and that made the error go away (TypeError: Cannot read properties of null (reading 'Symbol(traceUrl)')). 

But the extension still says "No tests have been found in this workspace yet".

I'm tested in on a fresh repo that was made using npm init playwright@latest

node v18.16.1
npm 9.5.1
ubuntu 22.04.2
vscode 1.80.2
extension v1.0.15 + the #364 patch
